### PR TITLE
Add janus setup instructions

### PR DESCRIPTION
**Pull request by rcuza**
This change adds instructions for adding this repo to janus.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ Set up [Vundle](https://github.com/VundleVim/Vundle.vim) then add `Plugin 'cespa
 
 Set up [vim-plug](https://github.com/junegunn/vim-plug). In your .vimrc, between the lines for `call plug#begin()` and `call plug#end()`, add the line `Plug 'cespare/vim-toml'`. Then run `:PlugInstall` from a fresh vim instance.
 
+### Janus
+
+Set up [janus](https://github.com/carlhuda/janus) then clone/submodule this repo into `~/.janus` and restart vim.
```
Comment (cespare) [inline README.md]: s/janus/Janus/

s/then/and then/ (or else add a comma; either way, get rid of the comma splice).

**Comment by rcuza** [comment]
@cespare - Thank you. I believe I've made the changes you requested. Thank you!

**Comment by cespare** [comment]
Thanks, but you changed "then" (correct) to "than" (incorrect). :P

I went ahead and did the merge/squash myself in ebd42fae64130816a4ef56728dcea110a32c65b8. Thanks for the contribution!

**Comment by rcuza** [comment]
LOL. 

I am working on a theory of conservation of bad grammar. I will have to include this as a data point.

Thank you for the assist.

**Pull request closed without merging**